docs/azure/events.md

```diff
@@ -1,4 +1,8 @@
-# Semantic Conventions for Azure Resource Log events
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Events
```

Comment:
Wouldn't "Events" here be too general? Or is it because it's inside `azure` that then it doesn't matter?

Reply:
Right. Currently it looks like this:

> <img width="165" alt="image" src="https://github.com/user-attachments/assets/60abba4f-cdc2-4454-a843-af6bb66fb604" />

With the change it'll look like this, withe "Events" in context of "Azur":

> <img width="104" alt="image" src="https://github.com/user-attachments/assets/847a283a-c907-483b-9967-19d6c2a91ece" />